articles.html

```diff
@@ -3,18 +3,243 @@
 
   <head>
     <title>US Constitution Articles</title>
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css" />
+    <script type="text/javascript" src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    <script type="text/javascript" src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+
     <!-- Getting fonts and fontawesome icons -->
     <link rel='preconnect' href='https://fonts.googleapis.com'>
     <link rel='preconnect' href='https://fonts.gstatic.com' crossorigin>
     <link href='https://fonts.googleapis.com/css2?family=Lato&family=Roboto+Slab&display=swap' rel='stylesheet'>
     <script src='https://kit.fontawesome.com/4bfa365d66.js' crossorigin='anonymous'></script>
     <meta charset='UTF-8'>
     <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+
     <link rel='stylesheet' href='assets/style.css'>
+    <script type="text/javascript" src="assets/script.js"></script>
   </head>
 
   <body>
+    <h1 class="head">The 10 Articles of the US Constitution</h1> 
+    <div id="articles">
+      <!--------------------------Article 1------------------------------>
+      <h3>Article I</h3>
+      <div>
+        <p><b>Section 1</b></p>
+        <p>
+          All legislative Powers herein granted shall be vested in a Congress of the United States, which shall consist of a Senate and House of Representatives.
+        </p>
```

Comment:
I would use <section> tags to separate each article of the constitution, otherwise this looks good